source/nodejs/adaptivecards-designer/src/open-image-dialog.ts

```diff
@@ -0,0 +1,351 @@
+    private _predictedCardJSON: predictedCardJSON;
+    private fetchManager: FetchManager;
+    private isTemplateOptionChecked: boolean = false;
+    private allowedImageSize = 2; // In MB
+
+    constructor() {
+        super();
+        this.fetchManager = new FetchManager();
+    }
+
+    private setContent(element: HTMLElement) {
+        this._renderedElement.appendChild(element);
+    }
+
+    private removeContent(element: HTMLElement) {
+        this._renderedElement.removeChild(element);
+    }
+
+    private renderTitle(): HTMLElement {
+        const title = document.createElement("div");
+        title.className = "acd-image-title";
+        title.innerText = "Upload your own card design";
+        return title;
+    }
+
+    private renderTitleContent(): HTMLElement {
+        const content = document.createElement("div");
+        content.className = "acd-image-title-content";
+        content.innerText =
+            "Design your card and upload it as a png or jpeg, and watch it magically convert to a functioning Adaptive Card";
+        return content;
+    }
+
+    private renderUploadedImage(): HTMLElement {
+        this._imageElement = document.createElement("img");
+        this._imageElement.className = "acd-image-placeholder";
+        this._imageElement.setAttribute("src", this._uploadedImage);
+        return this._imageElement;
+    }
+
+    private renderUploadButton(type: string): HTMLElement {
+        this._buttonContainer = document.createElement("div");
+        this._buttonContainer.className = "acd-upload-button-container";
+        const fileNote = document.createElement("div");
+        fileNote.className = "acd-image-file-note";
+        fileNote.innerHTML =
+            "By using the upload feature you agree to our <a href='../privacy' target='_blank'><span class='link'> image usage policy</span></a>";
```

Comment:
Sorry for the late ask here, but can we make the privacy link settable using a property? So as a consumer I could do something like this: 

`ACDesigner.Pic2Card.privacyLink = "../myPath/privacy";`

The default could be ../privacy if that works.